### ISSUE TITLE
Enhancement: insert filename as compatible path

### ISSUE THREAD

**Issue by timmie** [spyder-ide/spyder#2641]
Enhancement idea:

# Rationale:
When doing data analysis, I often struggle with the way path names are defiend in python & windows.

Due to spaces and backslashes in the file names the scipts often return errors because they cannot find the files.

# Suggestion:

A context-menu entry for the file explorer or file picker with:

* `insert with absolute path here`
* `insert as relative path here`

and if needed, the editor can propse the import of `os` / `sys` packages.

This would be great helper and get the filepath issues out of way.

**Comment by ccordoba12** [comment]
@goanpeca, I think we could add an *Insert file path here* dialog, with the right back slashes and all on Windows.

**Comment by chends888** [comment]
Hi, i wanted to know if this issue is still open. I'd like to start working on this. Ty!

**Comment by ccordoba12** [comment]
Yes, this issue is still open and it'd be very cool to have! Thanks for offering to implement it.

Please do it against our master branch because `3.x` is in maintenance mode.
